PullRequestClosed/PullRequestClosed.test.ts

```diff
@@ -0,0 +1,150 @@
+import * as core from '@actions/core';
+import * as github from '@actions/github';
+import { PullRequestClosed } from './PullRequestClosed';
+import { LogTester } from '../tests/LogTester';
+import { jiraClientStub } from '../tests/JiraClientStub';
+import { createOctokitRestStub } from '../tests/OctokitRestStub';
+
+async function runAction(title: string, user: string = 'test-user') {
+  process.env['INPUT_JIRA-PROJECT'] = 'KEY';
+  const action = new PullRequestClosed();
+  (action as any).jira = jiraClientStub;
+  (action as any).rest = createOctokitRestStub(title, "", user);
```

Comment:
You can now use the trick from GHA-104
```suggestion
  const action = new PullRequestClosed() as PullRequestClosed & OctokitActionStub;
  action.jira = jiraClientStub;
  action.rest = createOctokitRestStub(title, "", user);
```